src/System.Runtime.Extensions/pkg/unix/System.Runtime.Extensions.pkgproj

```diff
@@ -11,6 +11,11 @@
     <ProjectReference Include="..\..\src\System.Runtime.Extensions.csproj">
       <OSGroup>Unix</OSGroup>
     </ProjectReference>
+    <!-- AOT implementation comes from AOT package -->
+    <File Include="$(PlaceholderFile)">
+      <TargetPath>runtimes/aot/lib/uap10.1</TargetPath>
```

Comment:
OK, I will undo Unix-related parts.  I thought we had plans to enable aot for Unix as well.